src/utils/domUtils.ts

```diff
@@ -64,7 +64,7 @@ export module Utils {
       return selection;
     }
 
-    export function boxesOverlap(boxA: ClientRect, boxB: ClientRect) {
+    export function clientRectOverlap(boxA: ClientRect, boxB: ClientRect) {
```

Comment:
`clientRectsOverlap()` makes more sense to me. Also avoids suggesting that it returns the overlap.